cpu/samd21/ldscripts/samr21x18a.ld

```diff
@@ -11,9 +11,9 @@
  * @{
  *
  * @file
- * @brief           Memory definitions for the SAMR21G18A
+ * @brief           Memory definitions for the SAMR21E18A and SAMR21G18A
  *
- * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author          Michael Andersen <m.andersen@cs.berkeley.edu>
```

Comment:
@immesys With the minor changes, I'd still consider @haukepetersen as author of this file.